Update

abdulrahmanshr75 opened the pull request:
Added the linters.
Notice that this project is from my last week and it is the full project of day 3 and day 4.

abdulrahman75 committed:
Update

Added the linters.

rotimiazeez commented on index.html:
```diff
@@ -0,0 +1,204 @@
+    <div class="main">
+      <div class="container">
+        <h1 class="Logo-heading">Welcome</h1>
+        <img class="icon" src="images/IconMenu.png" alt="Icon" />
```
- [ ] Kindly wrap the toolbar section in a `header` tag to make your code semantic and easily found by the search engine✔

rotimiazeez replied:
- [ ] kindly wrap the hamburger button in a `nav` tag as it'll contain other nav menus in the coming projects😊😊

rotimiazeez commented on index.html:
```diff
@@ -0,0 +1,204 @@
+            </li>
+          </ul>
+        </div>
+      </div>
```
- [ ] Kindly wrap this section in a `section` tag instead of a `div` tag as it makes your code more semantic and readable😉👍

rotimiazeez commented on index.html:
```diff
@@ -0,0 +1,204 @@
+        <h1 class="Logo-heading">Welcome</h1>
+        <img class="icon" src="images/IconMenu.png" alt="Icon" />
+        <div class="content">
+          <h2 class="heading">Hey There, I'am Abdulrahman Shrshar</h2>
```
- [ ] Kindly make sure this title is wrapped in an `h1` tag as it is the main header of the page😎😎

rotimiazeez commented on index.html:
```diff
@@ -0,0 +1,204 @@
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <ul class="social-icons">
+            <li>
+              <img class="icons" src="images/twitter.png" alt="twitter" />
+            </li>
+            <li>
+              <img class="icons" src="images/Linkedinicon.png" alt="Linkedin" />
+            </li>
+            <li><img class="icons" src="images/medium.png" alt="medium" /></li>
+            <li><img class="icons" src="images/github.png" alt="github" /></li>
+            <li>
+              <img class="icons" src="images/angellist.png" alt="angellist" />
+            </li>
```
- [ ] Kindly make each icon has a link (`a` tag) as they are a clickable link 🙏🏽🙏🏽

rotimiazeez reviewed (changes requested):
# Hi @abdulrahmanshr75 👋, 
![](https://media.giphy.com/media/3orieSeDRFY491jKwM/giphy.gif)
### Good job so far!👏👏
### There are some issues that you still need to work on to go to the next project but you are almost there! 🚀🚀
> Kindly make the suggested change and you are good to go🚀🚀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahman75 committed:
Changes

- wrap the toolbar section in a header tag
-  wrap the hamburger button in a nav tag
- wrap the div section in a section tag
- the title is wrapped in an h1 tag
-  icon has a link (a tag)

abdulrahman75 committed:
changes

Fixed linters
Fixed the a links in the social media in the first section ( note : it is not allowed to have an a tag inside of an ul tag)

lily-coder commented on README.md:
```diff
@@ -0,0 +1,25 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Abdulrahman's ؛ortfolio
```
- [ ] [OPTIONAL] Please fix the spelling error. You can also make this line a header to make it more visually appealing. Eg;
`# Abdulrahman's Portfolio`

lily-coder commented on README.md:
```diff
@@ -0,0 +1,25 @@
+
+- HTML
+- CSS
+
```
- [ ] [OPTIONAL] You can add the following section to help users, especially those not so familiar with coding to set up and view your work 
```
### Setup
To get a local copy up and running follow these simple example steps.
- Clone the repository to your computer using git clone [Git Clone Url](https://github.com/abdulrahmanshr75/Microverse-Portfolio.git)
- Cd into the folder
- Open index.html with live server
- Or you can Fork it instead, and create a pull request to suggest some changes
```

lily-coder commented on index.html:
```diff
@@ -0,0 +1,215 @@
+    <div class="main">
+      <header>
+        <nav>
+          <h1 class="Logo-heading">Welcome</h1>
```
- [ ] [REQUIRED] I think you should remove this `h1` from the nav tag and also make it an `h2` or `h3` tag because there is only supposed to be one h1 tag on each page😊. So it will be
```
<h2 class="Logo-heading">Welcome</h2>
<nav>
    <img class="icon" src="images/IconMenu.png" alt="icon" />
</nav>
```

lily-coder reviewed (changes requested):
Hi @abdulrahmanshr75 , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
![image](https://user-images.githubusercontent.com/63964646/144023114-c3f4b06a-6ddd-4e07-9cf6-a679b9235489.png)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahman75 committed:
changes

- fixed the spelling error.
- removed the h1 from the nav tag and made it an h2
- added the optional edit to readme file

VanessaAoki reviewed (approved):
## STATUS: APPROVED 🟢 
Hi @abdulrahmanshr75, 

Great job implementing all the changes required by the previous Code Reviewers, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

<img alt="approved" src="https://64.media.tumblr.com/87e0b6469c957bf896cdec1cab646acf/tumblr_inline_pk1bdakCCp1rpcr9a_500.gif">

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
